test/integration/cart_integration_test.rb

```diff
@@ -0,0 +1,20 @@
+require 'test_helper'
+
+class CartIntegrationTest < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+
+  test 'an unauthorized user can add an item to the cart' do
+    item = Item.create(title: 'coffee', description: 'black nectar of the gods', price: 1200)
+    visit "/items/#{item.id}"
+    click_link_or_button('Add to Cart')
+    within('#flash_notice') do
+      assert page.has_content?('Added to Cart')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.